src/Employer/Employer.Web/Controllers/DashboardController.cs

```diff
@@ -20,10 +20,15 @@ public DashboardController(ILogger<DashboardController> logger, DashboardOrchest
         }
 
         [HttpGet("dashboard", Name = RouteNames.Dashboard_Index_Get)]
-        public async Task<IActionResult> Index()
+        public async Task<IActionResult> Dashboard()
         {
             var employerDetail = (EmployerIdentifier)HttpContext.Items[ContextItemKeys.EmployerIdentifier];
             var vm = await _orchestrator.GetDashboardViewModelAsync(employerDetail);
+            
+            if (TempData.ContainsKey(TempDataKeys.DashboardErrorMessage))
+            {
+                ModelState.AddModelError(string.Empty, TempData[TempDataKeys.DashboardErrorMessage].ToString());
+            }
```

Comment:
Previously I've done TempData-to-ModleState stuff using an attribute like this. May be worth a butchers. http://piotrgankiewicz.com/2016/04/25/postredirectget-with-new-asp-net-5-mvc-6/

Reply:
Hmmm. Looks like a mare in Core. Not worth the faff I think